src/core/Akka.Remote/Endpoint.cs

```diff
@@ -239,6 +244,8 @@ public EndpointAssociationException(string msg)
             : base(msg)
         {
         }
+
+        public EndpointAssociationException(string msg, Exception inner) : base(msg, inner) { }
```

Comment:
This is a public class, but I added an overload to the constructor to make it possible to pass along an inner `Exception`